index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>themeWagon</title>
+		<!-- <link rel="stylesheet" href="../task-html-and-css-basics/reset.css" /> -->
+		<script
+			src="https://kit.fontawesome.com/02b621b881.js"
+			crossorigin="anonymous"
+		></script>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100..900;1,100..900&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="../task-html-and-css-basics/style.css" />
+	</head>
+	<body>
+		<header class="header">
+			<nav class="header__nav wrapper">
+				<div class="header__nav--logo">
+					<h1>nova</h1>
+				</div>
+				<div class="header__nav--menu" id="#" <a href="#"></a>
+					<a href="#team">The Team</a>
+					<a href="#pricing">Pricing</a>
```

Comment:
Pamiętaj o semantyce - tutaj zdecydowanie pasuje `nav`, dodatkowo osobiscie wolą strukturę `ul > li`, niż samo `a`, ale to nie błąd.